purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -58,7 +57,11 @@ def _post_products_without_supplier_message(self, products):
 
     def _create_purchase_order(self, seller_id):
         vals = {"partner_id": seller_id}
-        ctx = {"default_requisition_id": self.id}
+        if self.user_id:
+            uid = self.user_id.id
+        else:
+            uid = self.env.user.id
+        ctx = {"default_requisition_id": self.id, "default_user_id": uid}
```

Comment:
Please extract this change in a separate commit, because it is a new functionality.